sbt-scalajs-bundler/src/main/scala/scalajsbundler/Webpack.scala

```diff
@@ -6,6 +6,14 @@ import scalajsbundler.util.{Commands, JS}
 
 object Webpack {
 
+  def copyToWorkingDir(targetDir: File)(file: File): File = {
+    val copy = targetDir / file.name
+    if (copy.getAbsolutePath != file.getAbsolutePath && !copy.exists()) {
```

Comment:
Why are these tests necessary? Does that happen to have the same source and destination? In such a case maybe we should let the IO.copyFile method throw an error. Also, the second test will make it impossible to update a file that has been copied and then changed afterwards. Unless I’m missing something?